### DISCUSSION TITLE
Did Minio's backend storage format change?

### DISCUSSION THREAD

**Discussion by dstromberg** [minio/minio#15060] [Q&A] [answered]
Hi.

Did MinIO's S3 backend file layout change recently?

I have a (docker) containerized application that has been receiving files over MinIO S3, and exporting them through NGINX, via a pair of docker volumes.

This suddenly doesn't work anymore, and I'm wondering if there's an explanation for it here.  I'm getting 404's and 301's, where things were 200'ing.

Please advise.

Thanks!

**Comment by harshavardhana** [comment, marked as answer]
We do not support existing data anymore read about them here https://docs.min.io/minio/baremetal/installation/deploy-minio-single-node-single-drive.html#deploy-minio-as-single-node-single-drive
